src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledObjects/PooledHashSet`1.cs

```diff
@@ -16,7 +16,9 @@ namespace Microsoft.AspNetCore.Razor.PooledObjects;
 internal ref struct PooledHashSet<T>
 {
     private readonly ObjectPool<HashSet<T>> _pool;
+#pragma warning disable IDE0052
     private readonly int? _capacity;
+#pragma warning restore IDE0052
```

Comment:
This one makes me a bit sad, but I made the change locally to see why it's needed.

Reply:
Yah. I almost went with just `_ = _capacity;` in the method where this is used, instead.

Reply:
I _don't_ see why this is needed: perhaps a comment?

Reply:
FWIW, it's because the only thing using this is under `#if NET`, since `HashSet<T>.EnsureCapacity(...)` is only available on .NET.

Reply:
I'm adding "modern" into my comment because "only available on .NET" still sounds odd to me :P

Reply:
It will always sound odd. :wink: